rcore/bluetooth.c

```diff
@@ -257,7 +271,26 @@ inline uint8_t bluetooth_send(uint8_t *data, size_t len)
 void bluetooth_device_connected(void)
 {
     _connected = true;
+    protocol_set_current_transport_sender(bluetooth_send_data);
     connection_service_update(true);
+    
+    /* Forge an rx ping to set the transport, and to transmit real data
+     * over the link (a pong).  */
+    static const uint8_t appversion_req[] = {
+        0x00
+    };
+    RebblePacket pkt = packet_create_with_data(WatchProtocol_AppVersion, appversion_req, sizeof(appversion_req));
+    packet_send(pkt);
+
+#if 0    
```

Comment:
weirdly this is still something that is needed on snowy

Reply:
Ok, ifdef'ed for snowy.